pkg/common/resources/resources.go

```diff
@@ -751,6 +768,47 @@ func StrictlyGreaterThanOrEquals(larger, smaller *Resource) bool {
 	return true
 }
 
+// StrictlyGreaterThanOnlyExisting Return true if all quantities present or existing only in larger > smaller
+// Two resources that are equal are not considered strictly larger than each other.
+// Resource present in smaller but not in larger are not even considered.
+func StrictlyGreaterThanOnlyExisting(larger, smaller *Resource) bool {
```

Comment:
I think this signature should be `func (r *Resource) StrictlyGreaterThanOnlyExisting(smaller *Resource) bool`
With the description:
```
// StrictlyGreaterThanOnlyExisting returns true if all quantities for types in the defined resource are greater than
// the quantity for the same type in smaller.
// Types defined in smaller that are not in the defined resource are ignored.
// Two resources that are equal are not considered strictly larger than each other.
```
For consistency we should add and implement: if smaller is nil or empty all quantities in the defined resource must be greater than 0 (see below comment)